react/components/results.js

```diff
@@ -20,6 +20,9 @@ const ResultsComponent = React.createClass({
       if(block.thumb) {
         style.backgroundImage = "url(https://gist.githubusercontent.com/" + block.userId + "/" + d._id + "/raw/" + block.thumb + "/thumbnail.png)"
       }
+      else if(block.thumbBase64){
+        style.backgroundImage = "url(data:image/png;base64," + block.thumbBase64 + ")"
```

Comment:
is there a way we could host the thumbs on S3 or a service? that way we could get them by URL instead of download the blobs for each one. This would allow the browser to cache on a per-thumbnail basis too.